src/index.html

```diff
@@ -8,7 +8,205 @@
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <div id="header" class="header-wrapper">
+        <ul id="main-nav">
+          <li class="main-nav-item"><a href="">Home</a></li>
+        </ul>
+        <div id="nav-side-btn">
+          <button>Download my full resume</button>
+        </div>
+      </div>
+
+      <div id="banner1" class="banner1-wrapper">
+        <div id="banner1-cv-title" class="banner1-cv-title">
+          <div class="banner1-cv-title-name"><span>John Roberts</span></div>
+          <div class="banner1-cv-title-position">
+            <span>BUSINESS EXPERT</span>
+          </div>
+        </div>
+      </div>
+
+      <div id="banner2" class="banner2-wrapper">
+        <div if="banner2-personal-info" class="banner2-personal-info">
+          <div class="avatar-container">
+            <img src="" alt="photo" />
+          </div>
+          <div id="personal-info-Carousel" class="personal-info-wrapper">
+            <ul class="carousel-container">
+              <li class="carousel-cards" id="personal-info-aboutMe">
+                <h2 class="personal-info-cards-title">ABOUT ME</h2>
+                <div class="social-media-btn-group">
+                  <img class="social-media-btn" src="" alt="linkedin" />
+                  <img class="social-media-btn" src="" alt="twitter" />
+                  <img class="social-media-btn" src="" alt="facebook" />
+                </div>
+                <div id="personal-info-aboutMe-left-p">
+                  <p>
+                    I'm a paragraph. Click here to add your own text and edit
+                    me. It’s easy. Just click “Edit Text” or double click me to
+                    add your own content and make changes to the font. Feel free
+                    to drag and drop me anywhere you like on your page. I’m a
+                    great place for you to tell a story and let your users know
+                    a little more about you.
+                  </p>
+                </div>
+                <div id="personal-info-aboutMe-right-p">
+                  <p>
+                    I'm a paragraph. Click here to add your own text and edit
+                    me. It’s easy. Just click “Edit Text” or double click me to
+                    add your own content and make changes to the font. Feel free
+                    to drag and drop me anywhere you like on your page. I’m a
+                    great place for you to tell a story and let your users know
+                    a little more about you.
+                  </p>
+                </div>
+              </li>
+
+              <li class="carousel-cards" id="personal-info-hobbies">
+                <h2 class="personal-info-cards-title">HOBBIES & INTERESTS</h2>
+                <ul class="personal-info-hobbies-list">
+                  <li class="personal-info-hobbies-item">
+                    <img
+                      class="personal-info-hobbies-item-icon"
```

Comment:
Don't need to add class name to every tags. The class name is used for easily locating an element. Normally, if the parent tag has a class name, then the children's class names can be removed.